mlflow/ml-package-versions.yml

```diff
@@ -963,6 +985,8 @@ groq:
     maximum: "0.24.0"
     requirements:
     run: pytest tests/groq
+    # Run integration test with the lightweight tracing SDK
```

Comment:
nit: the field name feels self-explainatory, so we can probably skip this comment